gptables/examples/survey_data.py

```diff
@@ -0,0 +1,101 @@
+"""
+Labour market overview, UK: December 2020 - Real Survey Data Example
+--------------------------------------------------------------------
+
+This example demonstrates how to replicate the Labour Market overview accessible 
+example found at https://analysisfunction.civilservice.gov.uk/policy-store/further-resources-for-releasing-statistics-in-spreadsheets/
+"""
+
+import gptables as gpt
+import pandas as pd
+import numpy as np
+from pathlib import Path
+
+## Read data and arrange
+parent_dir = Path(__file__).parent
+
+labour_market_data = pd.read_csv(parent_dir / "survey_data.csv")
+labour_market_data.dropna(axis=0, how="all", inplace=True)
+labour_market_data.dropna(axis=1, how="all", inplace=True)
```

Comment:
As we are using this as a training example now, please can you add a comment to explain why we're doing this